Source/Core/InputCommon/CMakeLists.txt

```diff
@@ -124,6 +124,9 @@ elseif(APPLE)
     ${FORCEFEEDBACK_LIBRARY}
     ${IOK_LIBRARY}
   )
+  target_compile_options(inputcommon PRIVATE
+    -fobjc-arc
```

Comment:
Just to confirm, there are no other source files in inputcommon that will be affected by enabling ARC?

Reply:
In general, turning on arc is pretty safe, since it turns all the old mrc stuff into compiler errors